docs/preprocessing/functional-connectivity.md

```diff
@@ -0,0 +1,67 @@
+## Computing functional connectivity
+First, make sure that the preprocessed fMRI data are available as `derivatives` in your dataset.
+An example of data structure should be as follows:
+```
+├── derivatives
+│   ├── fmriprep-23.1.4
+│   │   ├── dataset_description.json
+│   │   └── sub-pilot
+│   │       ├── anat
+│   │       ├── figures
+│   │       └── ses-15
+```
+
+Then, the functional connectivity matrices can be computed using the `compute_fc.py` script.
+The simplest call of the script only needs a derivative dataset (usually from *fMRIPrep*).
+Following the above data structure, it would be called as follows:
+``` bash
+python compute_fc.py path_to_dataset/derivatives/fmriprep-23.1.4
+```
+
+???+ info "Default call of the `compute_fc.py` script"
+    When using the default options, the pipeline will (in this order and for all functional tasks):
+    
+    - [ ] Fetch the [DiFuMo](https://doi.org/10.1016/j.neuroimage.2020.117126) atlas (64 dimensions)
```

Comment:
Can we use *TemplateFlow* for this?

Reply:
That would be an idea, I opened #212 to do that.